apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -705,6 +696,25 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                     serviceFee: Big('0.0'),
                     implicitFee: Big('0.0'),
                 })
+
+                const amountToPay = Big(totalAmount.amountWithoutExplicitFee)
+                    .add(Big(totalAmount.explicitServiceCharge))
+                    .add(Big(totalAmount.explicitFee))
+                if (acquiringIntegration.minimumPaymentAmount && Big(amountToPay).lt(acquiringIntegration.minimumPaymentAmount)) {
+                    throw new GQLError({
+                        ...ERRORS.PAYMENT_AMOUNT_LESS_THAN_MINIMUM,
+                        messageInterpolation: { minimumPaymentAmount: Big(acquiringIntegration.minimumPaymentAmount).toFixed(2) },
+                    }, context)
+                }
```

Comment:
again

Reply:
we need to remove copy-paste from all mutations. It is not the goal of this PR